docs/src/pages/components/switches/switches.md

```diff
@@ -57,6 +57,47 @@ Here are some examples of customizing the component. You can learn more about th
 
 🎨 If you are looking for inspiration, you can check [MUI Treasury's customization examples](https://mui-treasury.com/styles/switch).
 
+## Unstyled switches
+
+The switch also comes with an unstyled version. It's ideal for doing heavy customizations and minimizing bundle size.
+
+```jsx
+import SwitchUnstyled from '@material-ui/unstyled/SwitchUnstyled';
+```
+
+The `SwitchUnstyled` component provides default components and assigns CSS classes you can style entirely on your own.
+You are free to choose any styling solution - plain CSS classes, a CSS framework, Emotion, etc.
+It is also possible to replace these default components by other HTML elements or custom components.
+
+There are three components you can override by the `components` prop: `Root`, `Thumb` and `Input`. Each one's props can be set using the `componentsProps` object.
+
+{{"demo": "pages/components/switches/UnstyledSwitches.js"}}
+
+### Recreation of Material-UI's Switch
+
+{{"demo": "pages/components/switches/UnstyledSwitchesMaterial.js"}}
+
+### useSwitch hook
```

Comment:
I like that we have demos with headless and unstyled API. It will allow us to look at the GA metrics and see which one gets more usage.

My gut intuition is that headless will be a bit more popular than the unstyled API in its current form. My reasoning:

- Developers coming from a different context would rather use the headless API to understand what's going on.
- Developers that want to go one layer down from MD can't use Unstyled directly because it's not the same component structure (because we didn't use Untyled in MD).

Reply:
> do you think it'll still be the case when we finally switch to use unstyled in MD?

I'm making wild bets here 😆.

Another interesting thing: I have heard multiple times design system leaders using our components with customizations that and they don't have the time to build great documentation, that they push the customization so they can work as much as possible with our documentation.